exporter/clickhouselogsexporter/exporter.go

```diff
@@ -79,6 +81,8 @@ const (
 		severity_text,
 		severity_number,
 		body,
+		body_json,
```

Comment:
can create v1 , v2 of insert tamplate and based on the config decide which one to use.